File: banda/IvanRC-banda-Java/src/main/java/modelsBandaMusicos/ICantante.java
package modelsBandaMusicos;

public interface ICantante {

    public void respirar();
    String tono = "";

    /**
     * función que sirve para recalcular el salario base del músico según su especialización
     * @param salario es el salario base a recalcular
     * @return el salario recalculado según la ocupación
     */
    public default Double recalcularSalario(Double salario){
        Double incremento = 1.4;
        return (salario*incremento);
    }
}

File: banda/IvanRC-banda-Kotlin/src/main/kotlin/modelsBandaMusicos/ICantante.kt
package modelsBandaMusicos

interface ICantante {
    open fun respirar()
    val tono: String

    /**
     * función que sirve para recalcular el salario base del músico según su especialización
     * @param salario es el salario base a recalcular
     * @return el salario recalculado según la ocupación
     */
    fun recalcularSalario(salario: Double): Double{
        val incremento = 1.4
        return (salario*incremento)
    }
}